.evergreen/config.yml

```diff
@@ -310,6 +531,51 @@ tasks:
           TOPOLOGY: "sharded_cluster"
       - func: "run tests"
 
+  - name: "test-4.4-standalone"
+    tags: ["4.4", "standalone"]
+    commands:
+      - func: "bootstrap mongo-orchestration"
+        vars:
+          MONGODB_VERSION: "4.4"
+          TOPOLOGY: "server"
+      - func: "run tests"
+
+  - name: "test-4.4-replica_set"
+    tags: ["4.4", "replica_set"]
+    commands:
+      - func: "bootstrap mongo-orchestration"
+        vars:
+          MONGODB_VERSION: "4.4"
+          TOPOLOGY: "replica_set"
+      - func: "run tests"
+
+  - name: "test-4.4-sharded_cluster"
+    tags: ["4.4", "sharded_cluster"]
+    commands:
+      - func: "bootstrap mongo-orchestration"
+        vars:
+          MONGODB_VERSION: "4.4"
+          TOPOLOGY: "sharded_cluster"
+      - func: "run tests"
+
+  - name: "test-4.4-aws-auth"
+    # "4.4" explicitly left off to keep this out of the generic matrix
+    tags: ["aws-auth"]
+    commands:
+      - func: "bootstrap mongo-orchestration"
+        vars:
+          ORCHESTRATION_FILE: "auth-aws.json"
+          MONGODB_VERSION: "4.4"
+          AUTH: "auth"
+          TOPOLOGY: "server"
+      - func: "add aws auth variables to file"
```

Comment:
does the test binary need to be recompiled to get the new environment variables? I remember something about that being true for the driver but I'm not sure.

Reply:
Nope, not anymore. That was because we used the [`option_env!](https://doc.rust-lang.org/std/macro.option_env.html) macro, which has a nicer API in that it returns an option rather than a Result but has the downside of resolving the variable at compile time. We now use [`std::env:var`](https://doc.rust-lang.org/std/env/fn.var.html), which annoyingly returns a Result, but it resolves the variable at runtime. See [here](https://github.com/mongodb/mongo-rust-driver/blob/5970f98c145ddcdf0988b6dfe2b684f10c8ddbdc/src/test/mod.rs#L23) for where we use it.